blocks/init/src/Blocks/components/card/card.php

```diff
@@ -16,10 +16,11 @@
 $blockClass = $attributes['blockClass'] ?? '';
 $selectorClass = $attributes['selectorClass'] ?? $componentClass;
 
-$unique = Components::getUnique();
+$unique = $attributes['uniqueWrapperId'] ?? Components::getUnique();
```

Comment:
I think @volfkarlo suggested we modify the `getUnique()` function to include an optional parameter `$wrapperId` that we can pass. In this case, the `$unique` would be modified to:

```php
$unique = Components::getUnique($attributes['uniqueWrapperId]);
```

Reply:
I will update this in the next pr